trulens_eval/trulens_eval/app.py

```diff
@@ -816,6 +816,11 @@ def main_input(
         `bindings`.
         """
 
+        if bindings is None:
```

Comment:
what is this for?

Reply:
An extra sanity check in case this method gets called on a wrapped method for whom Signature.bind failed.